client/src/util/fetchFromAPI.test.js

```diff
@@ -1,6 +1,16 @@
-import { expect, test } from "@jest/globals";
+import { expect, test, it } from "@jest/globals";
 import { queryAPI } from "./fetchFromAPI";
+//reset the fetch mock so the previous tests fon't interfere with current tests
+beforeEach(() => {
+  fetch.resetMocks();
+});
 
+it("mocks the fetch request", async () => {
+  fetch.mockResponseOnce(JSON.stringify({ foodMenu: { title: "potato" } }));
+  const response = await queryAPI();
```

Comment:
Should this call to `queryAPI` have an `options` parameter?

Reply:
Yes passed the `options` parameter to `queryAPI`
<img width="406" alt="Screen Shot 2023-02-01 at 10 02 11 AM" src="https://user-images.githubusercontent.com/34128735/216079843-c2e5289f-914f-46ad-8638-d05e6c871587.png">